.github/workflows/scorecard.yml

```diff
@@ -35,12 +35,12 @@ jobs:
         uses: actions/setup-node@v4.0.2
 
       - name: "Checkout code"
-        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

Comment:
We may want to update this to 4.x.x soon. GitHub is [deprecating actions that use node 16](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/).